lib/use-cases/share-plan.test.js

```diff
@@ -1,35 +1,32 @@
 import SharePlan from 'lib/use-cases/share-plan';
-import { nanoid } from 'nanoid';
-jest.mock('nanoid');
 
 describe('Share plan', () => {
-  nanoid.mockImplementation(() => 'random_string');
-
   const planGateway = {
     save: jest.fn()
   };
   const smsGateway = { sendMessage: jest.fn() };
 
   const authHeader = 'bearer token';
 
-  it('saves a new customer token', async () => {
+  it('updates the token in the database after sharing it', async () => {
     planGateway.get = jest.fn(() => ({
-      customerTokens: []
+      customerTokens: [{ token: 'abc', sharedDate: null }]
     }));
+
     const sharePlan = new SharePlan({ planGateway, smsGateway });
     const collaborator = { number: '123' };
 
     const expectedResult = {
-      createdDate: expect.anything(),
-      token: 'random_string'
+      sharedDate: expect.stringContaining(''),
```

Comment:
If you don't care about the date value, you can do this:

```suggestion
      sharedDate: expect.any(String),
```

Reply:
Omg, thank you, I spent too much time on this one :D